<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureShow</class>
 <widget class="QWidget" name="PictureShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="OLEDShow">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>191</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="PictureShowStatic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="StartTransfer">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>200</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始显示</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="UpgradProgressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="OLEDModeShowSet">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>模式设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="BarLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>53</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>显示：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectShowBinFile">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>143</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
